enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -0,0 +1,100 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static admin_urls %}
+
+{% block extrastyle %}
+<!-- TODO: find a way to extract it into a file and make edx-platform find that file -->
```

Comment:
@e-kolpakov you  can create a directory named 'static\enterprise' (use 'enterprise' prefix so that your files do not clash with existing files in edx-platform) inside `enterprise` app and put your static files in that directory, those static files can be accessed via django's static template tag 
`<script type="text/javascript" src="{% static 'enterprise/js/test.js' %}"></script>`
or
`<link type="text/css" rel="stylesheet" href="{% static 'enterprise/css/test.css' %}">`.
edx-platform will be able to find those file because of `django.contrib.staticfiles.finders.AppDirectoriesFinder` [here](https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1203).